db/migrate/20221219230651_add_product_year_to_intakes.rb

```diff
@@ -0,0 +1,7 @@
+class AddProductYearToIntakes < ActiveRecord::Migration[7.0]
+  def change
+    # Safe enough for migrations since in Dec 2022 there aren't a lot of reads & writes on the web app.
+    add_column :intakes, :product_year, :integer, null: false, default: 2022
```

Comment:
also need to add filterable_product_year and assure it gets denormalized

and maybe some indices on all these sortable client attributes, so (filterable_product_year, last_contacted_at_or_whatever)